tests/functional/harris/harris_2d.py

```
@@ -142,6 +142,9 @@ def vthz(x, y):
     ph.FluidDiagnostics(
         quantity="density", write_timestamps=timestamps, population_name="protons"
     )
+    ph.FluidDiagnostics(
+        quantity="momentum_tensor", write_timestamps=timestamps, population_name=pop
```

Ruff (0.11.9)

Undefined name `pop`

(F821)